subsys/bluetooth/audio/ascs.c

```diff
@@ -883,27 +883,33 @@ NET_BUF_SIMPLE_DEFINE_STATIC(ase_buf, CONFIG_BT_L2CAP_TX_MTU);
 static void ase_process(struct k_work *work)
 {
 	struct bt_ascs_ase *ase = CONTAINER_OF(work, struct bt_ascs_ase, work);
+	struct bt_audio_ep *ep = &ase->ep;
+	const struct bt_audio_iso *audio_iso = ep->iso;
+	struct bt_audio_stream *stream = ep->stream;
+	const uint8_t ep_state = ep->status.state;
 
-	BT_DBG("ase %p, ep %p, ep.stream %p", ase, &ase->ep, ase->ep.stream);
+	BT_DBG("ase %p, ep %p, ep.stream %p", ase, ep, stream);
 
 	if (ase->ascs->conn != NULL &&
 	    ase->ascs->conn->state == BT_CONN_CONNECTED) {
-		ascs_ep_get_status(&ase->ep, &ase_buf);
+		ascs_ep_get_status(ep, &ase_buf);
 
-		bt_gatt_notify(ase->ascs->conn, ase->ep.server.attr,
+		bt_gatt_notify(ase->ascs->conn, ep->server.attr,
```

Comment:
Have a local variable `conn` for `ase->ascs->conn`  as it is dereferenced multiple times.

```suggestion
		bt_gatt_notify(conn, ep->server.attr,
```